### PR TITLE
Parallel tests docs

### PR DESCRIPTION
**Pull request by chrisarcand**
Some extra help to point people at dealing with parallel_tests. Includes some little tips we've picked up since first presenting about this at the ManageIQ summit.

@miq-bot assign @Fryguy 
cc/ @jrafanie

### CODE REVIEW

**Comment by Fryguy** [comment]
:+1: < How you like that speed!

File: developer_setup/running_test_suites.md [hunk 1]
```diff
@@ -19,9 +20,84 @@ rake test:automation:setup       # Setup environment for automation specs
...
+  made some pretty drastic changes to role checking code having to do with a method named `#role_allows`. To quickly run specs that
+  *might* touch this code, I can pass in all tests that mention it directly:
+
+  ```
+  $ parallel_rspec `git grep -l role_allows spec`
+  ```
+
+* Each core requires a certain amount of overhead involved with loading all of the application code, etc. Always running the maximum number
+  of cores available isn't always beneficial to fast tests. If you're running a subset of tests and parallel_tests reports that you're running
+  a very small number of specs across each core, try using fewer cores doing more work instead. You can specify the number view the `-n` option.
```
Comment (chrisarcand) [inline developer_setup/running_test_suites.md]: Typo

**Comment by jrafanie** [comment]
👍 Nice @chrisarcand

**Pull request merged**